packages/home-service/src/types.d.ts

```diff
@@ -1,6 +1,19 @@
 declare module '*.graphql';
 declare module '*.json';
 
+type UserType = {
+    name: string;
+    title: string;
+    uid: string;
+    rhatUUID: string;
+    isActive ?: boolean;
+    memberOf: string[];
+    apiRole: string,
+    createdBy: string;
+    createdOn: Date;
+    updatedBy: string;
+    updatedOn: Date;
+  }
```

Comment:
If possible avoid this duplicate type definitions

Reply:
Removed it

Reply:
I would then have to `import '../../user-service/src/types';`

Reply:
Is importing a standard way to do it?

Reply:
And if I do so it gives an error for the HomeType